### DISCUSSION TITLE
how to add read.me

### DISCUSSION THREAD

**Discussion by fedorkanan** [community/community#64937] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

help me whit my profile

**Comment by naserianali** [comment]
add readme.md to your main repository

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
